app/institutions/dashboard/controller.ts

```diff
@@ -3,13 +3,14 @@ import { alias } from '@ember/object/computed';
 import moment from 'moment';
 
 import Controller from '@ember/controller';
+import { InstitutionsDashboardModel } from 'ember-osf-web/institutions/dashboard/route';
 import InstitutionModel from 'ember-osf-web/models/institution';
 import InstitutionDepartmentModel from 'ember-osf-web/models/institution-department';
 import InstitutionSummaryMetricModel from 'ember-osf-web/models/institution-summary-metric';
 import InstitutionUserModel from 'ember-osf-web/models/institution-user';
 
 export default class InstitutionsDashboardController extends Controller {
-    @alias('model.taskInstance.value') modelValue!: any;
+    @alias('model.taskInstance.value') modelValue!: InstitutionsDashboardModel;
```

Comment:
a little more type safety would be nice: this (and all the aliases based on it) should have `?` instead of `!`, since they're all undefined until/unless the task instance succeeds